modules/olm-overview.adoc

```diff
@@ -3,16 +3,34 @@
 // * operators/understanding/olm/olm-understanding-olm.adoc
 // * operators/operator-reference.adoc
 
```

Comment:
Want to add the new assembly location so folks know not to mess it up later :wink: 
```suggestion
// * installing/cluster-capabilities.adoc

```

Reply:
Done